sdk/ai/azure-ai-client/azure/ai/client/aio/operations/_patch.py

```diff
@@ -98,23 +98,23 @@ async def get_embeddings_client(self) -> "EmbeddingsClient":
 
         return client
 
-    async def get_azure_openai_client(self) -> "AzureOpenAI":
+    async def get_azure_openai_client(self) -> "AsyncAzureOpenAI":
```

Comment:
I discussed this with Marko today that we were wondering if these methods are better named
as create_xxx instead of get_xxx. The reason is that these are creating the instance of client.
Now if it is get, the user might think, it would get the same instance every time the getter is called.
With create it is describing better what is actually done.

Reply:
Good question. I'll bring this up with Johan. Maybe also put a comment in the apiview.dev asking this question. If we decide to change it, I'll do it in a separate PR.